### ISSUE TITLE
Add Certificate Authority (CA) bundle on HTTPS server options

### ISSUE THREAD

**Issue by rodolphovfreire** [tlaverdure/laravel-echo-server#201]
On Chrome Version 60.0.3112.90 (Official Build) (64-bit), the WSS connection is being refused on HTTPS connection. 

> NET::ERR_CERT_AUTHORITY_INVALID

The issue is that the larave-echo-server does not have an third parameter.

```
var https_options = {
 **ca: fs.readFileSync("/path/to/mydomain.ca-bundle"),**
 key: fs.readFileSync("/path/to/server.key"),
 cert: fs.readFileSync("/path/to/mydomain.crt")
};
```

I guess it should be added...

**Comment by tlaverdure** [comment]
I'm running the same version of Chrome with no issues. Let's see if this is reported by others.

Thanks

**Comment by meijdenmedia** [comment]
+1
This issue is indeed only related if you access it on port 6001.

**Comment by Pikumi** [comment]
The pull request #204 should fix this problem.

**Comment by tlaverdure** [comment]
This should be available now at 1.3.1
